language/language_samples.rb

```diff
@@ -12,58 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def sentiment_from_text project_id:, text_content:
+def sentiment_from_text text_content:
   # [START sentiment_from_text]
-  # project_id   = "Your Google Cloud project ID"
   # text_content = "Text to run sentiment analysis on"
 
   require "google/cloud/language"
 
-  language  = Google::Cloud::Language.new project: project_id
-  document  = language.document text_content
-  sentiment = document.sentiment
+  language = Google::Cloud::Language.new
+  response = language.analyze_sentiment content: text_content, type: :PLAIN_TEXT
+
+  sentiment = response.document_sentiment
 
   puts "Overall document sentiment: (#{sentiment.score})"
   puts "Sentence level sentiment:"
 
-  sentiment.sentences.each do |sentence|
+  sentences = response.sentences
+
+  sentences.each do |sentence|
     sentiment = sentence.sentiment
     puts "#{sentence.text}: (#{sentiment.score})"
   end
   # [END sentiment_from_text]
 end
 
-def sentiment_from_cloud_storage_file project_id:, storage_path:
+def sentiment_from_cloud_storage_file storage_path:
   # [START sentiment_from_cloud_storage_file]
-  # project_id   = "Your Google Cloud project ID"
   # storage_path = "Path to file in Google Cloud Storage, eg. gs://bucket/file"
 
   require "google/cloud/language"
 
-  language  = Google::Cloud::Language.new project: project_id
-  document  = language.document storage_path
-  sentiment = document.sentiment
+  language = Google::Cloud::Language.new
+  response = language.analyze_sentiment gcs_content_uri: storage_path, type: :PLAIN_TEXT
```

Comment:
The surface would look cleaner if `gcs_content_uri:` and `content:` were positional parameters and auto-detected the type of input either a gcs path or a string. 
Then the following:
```ruby
language.analyze_sentiment gcs_content_uri: storage_path, type: :PLAIN_TEXT
```

Would be:
```ruby
language.analyze_sentiment storage_path, type: :PLAIN_TEXT
# OR
language.analyze_sentiment content, type: :PLAIN_TEXT
```